flow.c

```diff
@@ -250,6 +253,19 @@ void flow_delete(struct flow *f)
                 thread_clear_flow_or_die(f->f_thread, f);
         }
 
+#ifdef WITH_TCPDIRECT
+        if (flow_thread(f)->opts->tcpd_gpu_pci_addr) {
+                cuda_flow_cleanup(f->f_mbuf);
+        } else if (flow_thread(f)->opts->tcpd_nic_pci_addr) {
+                struct tcpdirect_udma_mbuf *t_mbuf = (struct tcpdirect_udma_mbuf *)f->f_mbuf;
+
+                close(t_mbuf->buf_pages);
+                close(t_mbuf->buf);
+                close(t_mbuf->memfd);
+                close(t_mbuf->devfd);
```

Comment:
Hmm leaving the udmabuf support in this commit makes it look like udmabuf is supported, but if folks try it, it won't work yet, right?

I'm fine with leaving it in because removing it only to re-add it again properly is wasted effort, but lets work on udmabuf support ASAP then.

I don't want to forward neper to upstream reviewers that say 'oh udmabuf is supported, how do I run it?' only to explain that it's not actually supported lol.